.github/workflows/test_base.yml

```diff
@@ -15,10 +15,11 @@ jobs:
         with:
           filters: |
             base:
+              - pyproject.toml
               - sktime/base/**
   test:
     needs: detect
-    name: test base component
+    name:
```

Comment:
I did not know name can be empty, so thanks for showing. But is it a good thing? If there's no name, how shall we know which one is causing issues if a CI fails?

Reply:
based on the GUI, it shows with a red X